sdk/storage/azure-storage-blob/src/main/java/com/azure/storage/blob/URLParser.java

```diff
@@ -15,7 +16,7 @@
 /**
  * A class used to conveniently parse URLs into {@link BlobURLParts} to modify the components of the URL.
  */
-final class URLParser {
+public final class URLParser {
```

Comment:
Same here.

Reply:
Deleted this class and merged it into BlobURLParts